inc/saiport.h

```diff
@@ -236,6 +236,24 @@ typedef enum _sai_port_interface_type_t
 
     /** Interface type KR4 */
     SAI_PORT_INTERFACE_TYPE_KR4,
+
+    /** Interface type CAUI */
+    SAI_PORT_INTERFACE_TYPE_CAUI,
+
+    /** Interface type GMII */
+    SAI_PORT_INTERFACE_TYPE_GMII,
+
+    /** Interface type SFI */
+    SAI_PORT_INTERFACE_TYPE_SFI,
+
+    /** Interface type XLAUI */
+    SAI_PORT_INTERFACE_TYPE_XLAUI,
+
+    /** Interface type CUSTOM BEGIN */
+    SAI_PORT_INTERFACE_TYPE_CUSTOM_BEGIN,
+
+    /** Interface type CUSTOM END */
+    SAI_PORT_INTERFACE_TYPE_CUSTOM_END = SAI_PORT_INTERFACE_TYPE_CUSTOM_BEGIN,
```

Comment:
Can you change that to: SAI_PORT_INTERFACE_TYPE_CUSTOM_BASE =  0x10000000
this way it will follow scheme like in other enums

Reply:
Yes, i will do it shortly!

Reply:
@kcudnik done!